test/gurka/test_route_summary.cc

```diff
@@ -7,7 +7,10 @@
 #include <valhalla/baldr/graphreader.h>
 #include <valhalla/baldr/traffictile.h>
 
+#ifndef _MSC_VER
```

Comment:
This should probably be `_WIN32`. I think I copied that ifdef some and existin ifdef for this header.